opentelemetry-api/src/opentelemetry/util/_time.py

```diff
@@ -12,16 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from logging import getLogger
 from sys import version_info
 
 if version_info.minor < 7:
+    getLogger(__name__).warning(  # pylint: disable=logging-not-lazy
```

Comment:
My suggestion for the warning:

```
You are using Python 3.%s. This version does not support timestamps with nanosecond granularity, and the Opentelemetry SDK will use millisecond granularity instead.

To use nanosecond granularity, please upgrade to Python 3.7 or newer.
```

Reply:
Ok, incorporated most of this. I still want to mention PEP 546 because the best explanation for this problem can be found there. :+1:

Reply:
>Opentelemetry SDK will use millisecond granularity instead.


```python
    def _time_ns():
         return int(time() * 1e9)
```

It's not milliseconds though? It is still nanos with less resolution, isn't it?

Reply:
I'd swap granularity with precision